SU2_CFD/src/numerics_direct_turbulent.cpp

```diff
@@ -534,6 +534,14 @@ void CSourcePieceWise_TurbSA::ComputeResidual(su2double *val_residual, su2double
 //  AD::SetPreaccIn(TurbVar_i[0]);
 //  AD::SetPreaccIn(TurbVar_Grad_i[0], nDim);
 //  AD::SetPreaccIn(Volume); AD::SetPreaccIn(dist_i);
+  
+//Added by Cakmakcioglu, for BC Transition Model 10/30/2106
+	bool transition    = (config->GetKind_Trans_Model() == BC);
```

Comment:
In fact, I am a beginner C++ user, so I am not sure if I understood this correctly.
I have defined "transition" in numerics_structure.hpp, and changed this part of the code as needed. Hope this it what you meant.